lib/core/src/components/index.ts

```diff
@@ -0,0 +1,3 @@
+import Button from './button'
```

Comment:
So this "Button" is importing the default "Plugin" export, not the named "Button" export?